Content: New alien species: the Incipias (#7230)
endless-sky/endless-sky: data/incipias/outfits.txt

	description "The secret of how these beams work is only known to a few scientists of the Incipias who are under the control of the state. Such knowledge, if made public, could have disastrous consequences."

effect "tractor beam impact"
	sprite "effect/tractor impact"
		"no repeat"
		"frame rate" 30
	"lifetime" 5
	"random angle" 2
	"velocity scale" -.005

outfit "Gas Class Shield"
	category "Systems"
	thumbnail "outfit/gas class shield"
	mass 60
	"outfit space" -60
	"shield generation" .1
	"energy generation" 4.2
	"heat generation" 5
	description "The shield technology is one of the most recent technologies of the Incipias. While the shield appears similar to a human shield device it works on a completely different base. In its core hydrogen is burned that provides energy for regeneration. But because scientists saw a more efficient use of this burning technology, the Incipias also made it the main power source of all of their ships."

outfit "Liquid Class Shield"
	category "Systems"
	thumbnail "outfit/liquid class shield"
	mass 80
	"outfit space" -90
	"shield generation" .25
	"energy generation" 6.1
	"heat generation" 6

> 	description "This more advanced shield generator burns a liquid instead of a gas. This raises the efficiency by a lot but also requires some extra structures which make this outfit only popular on larger ships."

Reviewer: ```suggestion
	description "This more advanced shield generator burns a liquid instead of a gas. This raises the efficiency by a lot but also requires some extra structures, making this outfit popular only on larger ships."
```

outfit "Solid Class Shield"
	category "Systems"
	mass 100
	"outfit space" -120
	"shield generation" .45
	"energy generation" 6
	"heat generation" 8
	description "This shield generator is pure efficiency. It burns its input in stages from solid in the center to a gaseous state in the outer hull. Sadly this efficiency requires a lot of adjusting systems to stabilize the process, and therefore only the heaviest warships use this generator."

outfit "Liquid Hydrogen Cell"
	category "Power"
	thumbnail "outfit/liquid hydrogen cell"
	mass 20
	"outfit space" -20
	"energy capacity" 2000
	description "This form of energy storage was discovered early by the Incipias because of the great amount of Hydrogen on their homeworld. Because the Incipias mostly access the energy at the time it is produced only a small storage is needed. That is why most of their ship have only one cell installed."

outfit "Heavy Machine Gun"
	category "Hand to Hand"
	cost 10000
	thumbnail "outfit/heavy machine gun"
	licenses
		"Hicemus Conflict"
	"capture attack" .55
	"capture defense" 1.5
	"unplunderable" 1

> 	description "When it comes to close combat, the Incipias are serious enemies. This brutal weapon may not be the best to invade another ship, but when it comes to defending the own ship, it is shredding the enemies inside the hallways."

Reviewer: ```suggestion
	description "When it comes to close combat, the Incipias are serious enemies. This brutal weapon may not be the best to invade another ship with, but when it comes to defending, it shreds enemies."
```

outfit "Plasma Discharger"
	category "Guns"
	mass 25

> 	"outfit space" -20

Reviewer 2: Is it deliberate that this doesn't use any weapon capacity?

Author: Ahem no
